tasks/commit.js

```diff
@@ -1,10 +1,9 @@
-"use strict";
```

Comment:
Don't remove the global `use strict`. We shouldn't turn files that were already in strict mode into sloppy mode.

I'd also expect it to be validated by ESLint so that all the Node.js-based files do have a global `"use strict";` pragma.

Reply:
Ok, done